### DISCUSSION TITLE
: fatal error C1083: Cannot open include file: 'io.h': No such file or directory

### DISCUSSION THREAD

**Discussion by GeoCalcul** [pypr/pysph#382] [General]
Hello,
I have just joined the group, and I'm happy to be here. I am currently using VS 2022 and encountering the following error:

C:\PythonSPH\pysph-master\env\lib\site-packages\Cython\Compiler\Main.py:369: FutureWarning: Cython directive 'language_level' not set, using 2 for now (Py2). This will change in a later release! File: C:\Users\STAND-~1\AppData\Local\Temp\tmpr0cloe_h\check_omp.pyx
  tree = Parsing.p_module(s, pxd, full_module_name)
check_omp.c
C:\Users\Stand-L7C24\AppData\Local\Programs\Python\Python310\include\pyconfig.h(59): fatal error C1083: Cannot open include file: 'io.h': No such file or directory
**********************************************************************
Unable to compile OpenMP code. Not using OpenMP.
**********************************************************************
Warning: passing language='c++' to cythonize() is deprecated. Instead, put "# distutils: language=c++" in your .pyx or .pxd file(s)

**Comment by nauaneed** [comment]
Could you check if you have installed Windows SDK along with MSVC? 
I believe `io.h` is provided by Windows SDK. So, if that's not installed, please do so.
